db/tests/columns/operations/test_select.py

```diff
@@ -118,9 +118,3 @@ def test_get_column_generated_default(engine_with_schema, col):
     ("'abcde'::CHAR(3)", False),
     ("'abcde'::CHAR(5)", False),
 ]
-
-
-@pytest.mark.parametrize("default_expr,is_dynamic", default_expression_test_list)
-def test_is_default_expr_dynamic(default_expr, is_dynamic):
-    default_clause = DefaultClause(text(default_expr))
-    assert _is_default_expr_dynamic(default_clause) is is_dynamic
```

Comment:
We need to bring back (rewrite) this test. In similar situations, you can consider skipping, so that there's a placeholder for the functionality. I'm currently working on related code, so I can reintroduce a test for this.

Reply:
Oh, this is being tested in the Postgres library, right?

Reply:
The function it calls no longer exists.

You're probably correct that we need a test for the SQL function, but there are some tests in the `mathesar/tests/test_column_api.py` file that make sure this is happening (E.g., the `test_column_list` test has at least one column with a dynamic default, and another with a static default).

@Anish9901 is currently working through a project to add tests that are missing for SQL functions.